CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+## 1.0.0 - 2024-05-10
+
+- [add] Use embedded-hal 1.0 (#36)
```

Comment:
```suggestion
- [change] Use embedded-hal 1.0 (#36)
```

Reply:
It could be considered a feature, not just a change. But `[change]` works as well.